packages/app/plugin/src/ios/appDelegate.ts

```diff
@@ -19,7 +19,7 @@ export function modifyObjcAppDelegate(contents: string): string {
     contents = contents.replace(
       /#import "AppDelegate.h"/g,
       `#import "AppDelegate.h"
-@import Firebase;`,
+#import <Firebase/Firebase.h>`,
```

Comment:
According to [the RNFB docs](https://rnfirebase.io/#configure-firebase-with-ios-credentials), I don't know if it shouldn't be `#import <Firebase.h>` 🤔

Reply:
I believe we use to advice using `@import Firebase` a long time ago, but we also switched, maybe that's the confusion?

Either way this reminded me of some previous issues, and in triage of those we made sure both ways work (or, worked - `@import` won't work now I understand that:
https://github.com/invertase/react-native-firebase/pull/4069

Also, I made this change myself recently in my demo script - which hadn't been updated from `@import` style yet, and I can say I know this is working as well with new style:
https://github.com/mikehardy/rnfbdemo/commit/cfed4d8f30eedb196410f3148a0d72257dcb1351#diff-3e27ae623577d0c559bd8ab973a0c34af8cc65ef9beb9868a9cdb029854f8078R33

Which is all to say: I'm pretty comfortable this will be backwards compatible (excellent!) and also I know for RN68 you need it as well